### ISSUE TITLE
Player confusion when slimes are fought using abberation.

### ISSUE THREAD

**Issue by Snowofcone** [pearuhdox/Cartographer-2.0#118]
Its not clear which slimes are the player's slime. Fix could be turning the slime purple to give it high contrast.

**Comment by pearuhdox** [comment]
Issue fixed in upcoming commit. Slimes have been fullbright boosted so they should be more distinguishable. Can adjust the color if necessary.
